wazimap_ng/config/common.py

```diff
@@ -338,6 +338,16 @@ def get_env_value(env_variable):
     ('public', 'Public'),
 )
 
+DATASET_CONTENT_TYPES = (
+    ('quantitative', 'quantitative'),
+    ('qualitative', 'qualitative')
```

Comment:
can we create a const somewhere so that these values can be used elsewhere in the code, e.g.
`QUANTITATIVE = "quantitative"`